src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Adding Layer 2s
+description: The policy we use when adding a layer 2 to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding layer 2s {#adding-layer-2}
+
+We want to make sure we list the best resources possible so users can navigate the layer 2 space in a safe and confident manner.
+
+Anyone is free to suggest adding a layer 2 on ethereum.org. If there's a layer 2 that we have missed, **please suggest it!**
+
+We currently list L2s on the following pages:
+
+- [Optimistic rollups](/developers/docs/scaling/optimistic-rollups/)
+- [Zero-knowledge rollups](/developers/docs/scaling/zk-rollups/)
+<!-- TODO: Add layer-2 page -->
+
+Layer 2 is a relatively new and exciting paradigm for Ethereum. We've tried to create a fair framework for consideration on ethereum.org but the listing criteria will change and evolve over time.
+
+## Information we require {#information-we-require}
+
+Layer 2 submissions will be assessed by the following criteria:
+
+**[Listing on L2BEAT](https://l2beat.com)**
+
+- In order to be considered, this project must be featured on L2BEAT. L2BEAT provides a great risk assessment of layer 2 projects that we lean on for evaluating L2 projects. **If the project is not featured on L2BEAT, we will not list them as an L2 on ethereum.org.**
+
+[Learn how to add your L2 project to L2BEAT](https://github.com/l2beat/l2beat/blob/master/CONTRIBUTING.md).
+
+**Is the project open source?**
+
+**What category of layer 2 is it?**
+
+- Optimistic rollup
+- Zero-knowledge rollup
+- State channel
```

Comment:
Should we explicitly call out things like validiums/volitions/plasmas here as currently not being considered for listing? Could help prevent confusion

Reply:
Hmm, I think this should be fine for now since the options are listed, but maybe we see how this goes and reassess?

Reply:
I agree @corwintines